assets/js/base/components/cart-checkout/order-summary/index.tsx

```diff
@@ -2,17 +2,23 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import { useContainerWidthContext } from '@woocommerce/base-context';
 import { Panel } from '@woocommerce/blocks-checkout';
 
 /**
  * Internal dependencies
  */
-import OrderSummaryItem from './order-summary-item.js';
+import OrderSummaryItem from './order-summary-item';
 import './style.scss';
+import { CartItem } from '../../../../types/type-defs/cart';
```

Comment:
You can use `import type` if you're only importing a type, also we have `@woocommerce/types` aliased for this